src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
To make code cleaner, I'd extract all properties from the person prop at this level, to avoid typing `person.` each time.